ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -159,15 +163,20 @@ private void SetSetting<T>(string key, T value)
             var values = ApplicationData.Current.LocalSettings.Values;
             values[key] = value;
 #else
-            var values = ConfigurationManager.AppSettings;
-            if (values[key] == null)
+            var configFile = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
```

Comment:
>OpenExeConfiguration [](start = 50, length = 20)

What's the need for moving from AppSettings to ExeConfiguration?  Does AppSettings not provide the correct defaults?